lib/operation.js

```diff
@@ -371,7 +383,7 @@ function median (size) {
  * @returns {Sharp}
  * @throws {Error} Invalid parameters
  */
-function blur (sigma) {
+function blur (sigma, precision) {
```

Comment:
Given `precision` is optional (and too, I suppose, is `sigma`), now might be the time to change the method signature to also accept an Object of optional properties here.

Reply:
Sounds good to me, to confirm you want to do a breaking change to: `blur(options)`: `{ sigma, precision }`

Reply:
Non-breaking please, if possible. If the first parameter is a number (rather than an Object) then we should be able to assume it is the value for `sigma`.